tests/CustomerCreditValidationTest.php

```diff
@@ -329,15 +329,15 @@ public function testUmlautConversion($schema)
         $xpathDoc->registerNamespace('sepa', 'urn:iso:std:iso:20022:tech:xsd:'.$schema);
         // Date is correctly coded
         $testNode = $xpathDoc->query('//sepa:InitgPty/sepa:Nm');
-        $this->assertEquals('Only A-Z without aeoeuessAeOeUe initiatingPartyName', $testNode->item(0)->textContent);
+        $this->assertEquals('Only A-Z without aeoeuessAEOeUE initiatingPartyName', $testNode->item(0)->textContent);
         $testNode = $xpathDoc->query('//sepa:Cdtr/sepa:Nm');
-        $this->assertEquals('Only A-Z without aeoeuessAeOeUe creditorName', $testNode->item(0)->textContent);
+        $this->assertEquals('Only A-Z without aeoeueSSAEOEUE creditorName', $testNode->item(0)->textContent);
         $testNode = $xpathDoc->query('//sepa:EndToEndId');
-        $this->assertEquals('Only A-Z without aeoeuessAeOeUe creditorName', $testNode->item(0)->textContent);
+        $this->assertEquals('Only A-Z without aeoeueSSAEOEUE creditorName', $testNode->item(0)->textContent);
         $testNode = $xpathDoc->query('//sepa:Dbtr/sepa:Nm');
-        $this->assertEquals('Only A-Z without aeoeuessAeOeUe debtorName', $testNode->item(0)->textContent);
+        $this->assertEquals('Only A-Z without aeoeueSSAEOEUE debtorName', $testNode->item(0)->textContent);
```

Comment:
Same as above the changes are wrong as they indicate change of behaviour of the former implementation